lib/msf/base/sessions/command_shell.rb

```diff
@@ -211,7 +165,7 @@ def shell_close()
   def cleanup
     if rstream
       if !@cleanup_command.blank?
-        shell_command_token(@cleanup_command, 0)
```

Comment:
Now we are able to include a default 10 second timeout rather than forcing '0' so that sessions don't hang 10 seconds after closing when cleanup_command is specified.